sklearn/feature_extraction/text.py

```diff
@@ -996,6 +996,12 @@ class TfidfTransformer(BaseEstimator, TransformerMixin):
     sublinear_tf : boolean, default=False
         Apply sublinear tf scaling, i.e. replace tf with 1 + log(tf).
 
+    Attributes
+    ----------
+    idf_ : numpy array of shape [n_features, 1]
```

Comment:
This class doesn't seem to have or use a `self.idf_` attribute. Do you mean `self._idf_diag`?